xml/snmp_sysdescr.xml

```diff
@@ -5452,16 +5452,18 @@ Copyright (c) 1995-2005 by Cisco Systems
   <!--======================================================================
                               SonicWall
    =======================================================================-->
-  <fingerprint pattern="^SonicWALL (.*?)\s+\(SonicOS \S+ (\d[^\)]+)\)\s*$">
+  <fingerprint pattern="^SonicWALL (\S+)\s+(\d+).*?\(SonicOS \S+ (\d[^\)]+)\)\s*$">
     <description>SonicWall - SonicOS Enhanced variant</description>
     <example>SonicWALL NSA 220 (SonicOS Enhanced 5.8.1.2-20o)</example>
     <example>SonicWALL TZ 215 wireless-N (SonicOS Enhanced 5.8.1.2-6o)</example>
     <example>SonicWALL TZ190 Enhanced (SonicOS Enhanced 3.6.0.4-30e)</example>
+    <example>SonicWALL SuperMassive 9200 (SonicOS Enhanced 6.2.5.3-35n)</example>
     <param pos="0" name="os.vendor" value="SonicWall"/>
     <param pos="0" name="os.device" value="Firewall"/>
     <param pos="0" name="os.family" value="SonicOS"/>
-    <param pos="1" name="os.product"/>
```

Comment:
Based on a review of SonicWall's offerings it looks like we should have a line that statically sets the `os.product` value. 

`<param pos="0" name="os.product" value="SonicOS"/>`

Reply:
Good call, changes made, tests pass